keras_lr_finder/lr_finder.py

```diff
@@ -48,7 +48,7 @@ def find(self, x_train, y_train, start_lr, end_lr, batch_size=64, epochs=1):
         num_batches = epochs * N / batch_size
         self.lr_mult = (float(end_lr) / float(start_lr)) ** (float(1) / float(num_batches))
         # Save weights into a file
-        self.model.save_weights('tmp.h5')
+        self.weights = self.model.get_weights()
```

Comment:
It doesn't have to be a class attribute. Can we just make it a local variable?

Reply:
Yeah, of course. I can arrange that.

Reply:
It is now an in-function variable called initial_weights.